src/lib.rs

```diff
@@ -234,7 +237,7 @@ impl<R: gimli::Reader> Context<R> {
             }
 
             res_units.push(ResUnit {
-                dw_unit,
+                dw_unit: Rc::new(dw_unit),
```

Comment:
Is is possible to avoid the `Rc` here?

Reply:
Returned a reference from `find_dwarf_unit` instead.